Components/Transforms/WeightedCombinationTransform/elxWeightedCombinationTransform.hxx

```diff
@@ -126,17 +126,6 @@ WeightedCombinationTransformElastix<TElastix>::WriteToFile(const ParametersType
   /** Call the WriteToFile from the TransformBase. */
   this->Superclass2::WriteToFile(param);
 
-  /** Write WeightedCombinationTransform specific things. */
-  xout["transpar"] << std::endl << "// WeightedCombinationTransform specific" << std::endl;
-
-  /** Write normalize-weights option */
-  std::string normalizeString = "false";
-  if (this->m_WeightedCombinationTransform->GetNormalizeWeights())
-  {
-    normalizeString = "true";
-  }
-  xout["transpar"] << "(NormalizeCombinationWeights \"" << normalizeString << "\" )" << std::endl;
-
```

Comment:
Please check that `WeightedCombinationTransform::WriteToFile` cannot be entirely removed!

Reply:
Update: "SubTransforms" is now added to the `WeightedCombinationTransform` parameter map (pull request https://github.com/SuperElastix/elastix/pull/391 commit ae9c0fd25aa52bd5e83456ad1129b13d1972a9d6) And this pull request is adjusted to simply remove the `WeightedCombinationTransform::WriteToFile` override.  😃